osf/migrations/0170_remove_outdated_contributor_permissions.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.15 on 2019-07-08 15:39
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('osf', '0169_merge_20190618_1429'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='contributor',
+            name='admin',
+        ),
+        migrations.RemoveField(
+            model_name='contributor',
+            name='read',
+        ),
+        migrations.RemoveField(
+            model_name='contributor',
+            name='write',
+        ),
+        migrations.RemoveField(
+            model_name='institutionalcontributor',
+            name='admin',
+        ),
+        migrations.RemoveField(
+            model_name='institutionalcontributor',
+            name='read',
+        ),
+        migrations.RemoveField(
+            model_name='institutionalcontributor',
+            name='write',
+        ),
+    ]
```

Comment:
This migration will need to be rebased before it's merged, but will hold off until we're about ready to put it on staging, as it might get out of date again.